framework/xds_flags.py

```diff
@@ -190,6 +190,12 @@
     help="Whether to enable GFE debug headers and what value to use.",
 )
 
+ENABLE_DUALSTACK = flags.DEFINE_bool(
+    "enable_dualstack",
+    default=False,
+    help=("Enable support for Dual Stack resources to the framework."),
```

Comment:
Remove redundant parentheses  

```suggestion
    help="Enable support for Dual Stack resources to the framework.",
```

Reply:
Done